test-requirements.txt

```diff
@@ -8,3 +8,4 @@ vcrpy-unittest
 openupgradelib @ git+https://github.com/OCA/openupgradelib.git
 unittest2 # For shopinvader test_controller, which inherits component
 odoo-test-helper
+odoo13-addon-queue-job @ git+https://github.com/OCA/queue.git@refs/pull/236/head#subdirectory=setup/queue_job
```

Comment:
@rousseldenis can you drop this? Then we can merge!

Reply:
@simahawk Done